cmd/adsysd/integration_tests/adsys_test.go

```diff
@@ -244,8 +244,8 @@ ad_backend: %s
 
 # SSSd configuration
 sssd:
-  config: testdata/sssd-configs/sssd.conf-example.com
-  cache_dir: %s/sss_cache
+    config: testdata/sssd-configs/sssd.conf-example.com
+    cache_dir: %s/sss_cache
```

Comment:
Yeah, those were tabs instead of spaces. I think yaml has a problem with that, because the code wasn't working.

Or it could just have been gofmt adding tabs instead of spaces in my pc... Now I'm not sure

Reply:
As on my other comment, let’s try to have those kind of changes separated. (from what I see on the source file, we were just using 2 spaces and now we are using 4, so it’s not tabs vs space)